Makefile

```diff
@@ -145,15 +153,19 @@ run-% : $(OBJDIR)/%
 	@tests/tap.sh $(<:build/%=%)
 
 test : $(tests:$(OBJDIR)/%=run-%) $(examples:$(OBJDIR)/%=run-%)
+tst : ;@$(MAKE) $(MFLAGS) V=$(V) test
+ctc-% : $(ctests);@$(foreach tst,$(ctests),$(tst) /cpu/$*;)
+ctg-% : $(ctests);@$(foreach tst,$(ctests),$(tst) /gpu/$*;)
```

Comment:
What need is this filling? I'd rather incorporate it into the test system that can produce TAP output. Also, anything less than obvious (and probably obvious stuff too) should be documented.